src/ensembl/src/content/app/browser/browser-nav/BrowserNavBarMain.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+
+import ChromosomeNavigator from 'src/content/app/browser/chromosome-navigator/ChromosomeNavigator';
+import BrowserNavBarRegionSwitcher from './BrowserNavBarRegionSwitcher';
+import { ReactComponent as CloseIcon } from 'static/img/shared/close.svg';
+
+import styles from './BrowserNavBarMain.scss';
+
+export enum Content {
```

Comment:
This is not component though. It's 1) an enum; 2) a named export, rather than a default one.

In this particular case I don't think I actually need to export this enum. I thought of using it in tests, but then didn't have a need for it. I'll remove the export keyword.

Reply:
Removed the export.